test/jasmine/tests/plots_test.js

```diff
@@ -369,7 +370,7 @@ describe('Test Plots', function() {
                 .then(done);
         });
 
-        afterEach(destroyGraphDiv);
+        afterAll(destroyGraphDiv);
```

Comment:
Oops, if we're doing `beforeAll` we'd better match it with an `afterAll`! The second test (`should resize...`) didn't consider that it might be running against a missing plot (`mainSvgs.length` was `0`) so was in fact a noop. The third test (`should update...`) I guess was OK, `gd` was still an object with properties even if it wasn't attached to the DOM...

Reply:
Nice find :microscope: